### DISCUSSION TITLE
Problem with ordering SKpico2040DAC from PCBWay

### DISCUSSION THREAD

**Discussion by quelo1972** [frntc/SIDKick-pico#82] [General]
Hi,
I have a problem with the order of SKpico2040DAC from PCBWay, when I add it to cart it ask to me "Please fill in the layer order from the top-side view to the bottom-side view according to the layer names in the Gerber file. If you're unsure, you can leave it blank for now." as you can see in attach
![Immagine 2024-09-23 123729](https://github.com/user-attachments/assets/6d930510-71a3-444e-b619-87bd8508f57d)
How must I populate that form?
Thanks in advance.

**Comment by frntc** [comment]
The easiest thing is: refer PCBWay to my order of the PCBs! That advice seemed to work so far :-)

**Comment by quelo1972** [comment]
otherwise I could buy it already assembled but from the suppliers whose links you provided in the description I can't understand if the fully integrated all-feature-variant ("SKpico2040DAC") is available or if it's simply the one with the integrated DAC. Can you help me order the SKpico2040DAC Full featured variant from one of the official suppliers? Excuse me for my bad english.

**Reply by frntc** [reply]
It will (hopefully) soon be available from restore-store.de first. So far they only have the variant where you need the Pi Pico (clone).

**Comment by quelo1972** [comment]
Cool! I didn't realize that the full version doesn't need the
Raspberry Pi Pico!!! We hope that it will soon be available already
assembled, because it is a bit complicated to solder!



Il Mar 24 Set 2024, 15:29 frntc ***@***.***> ha scritto:

**Comment by quelo1972** [comment]
Hi @frntc,
Since in your other projects, for example the RAD, you have also made the gerber file available on GitHub, could you please also provide the one for the the fully integrated all-feature-variant SKpico2040DAC?
Thanks in advance.

**Reply by frntc** [reply]
Please have a look [here](https://github.com/frntc/SIDKick-pico/discussions/3), where I'm explaining why I'm not planning to do this right now.

**Comment by quelo1972** [comment]
I fully understand your motives and I also agree with them! But PCBWay for
the RAD cost me a lot of shipping costs and taxes that their sum far
exceeded the cost of the 5 PCBs of which one was pre-assembled! I tried to
place the order for SK pico2040DAC, but I'm asked to order the layers but
I'm not able to provide it. If it were possible to know when restore store
will put up for sale SKpico2040DAC would help, unfortunately my C64 needs a
replacement for its failed SID!

Il Gio 10 Ott 2024, 22:53 frntc ***@***.***> ha scritto:
